helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -167,6 +167,24 @@ extraAttributes:
     # - k8s-app
     # - release
 
+  # Labels that will be collected from k8s namespaces (in case they are set)
+  # and added  as extra attributes to the telemetry in the following format:
+  # k8s.namespace.labels.<label_name>: <label_value>
+  namespaceLabels: []
+#    - dev
+#    - staging
+#    - production
+
+  # Annotations that will be collected from k8s pods (in case they are set)
+  # and added as extra attributes to the telemetry in the following format:
+  # k8s.pod.annotations.<label_name>: <label_value>
+  podAnnotations: []
+
+  # Annotations that will be collected from k8s namespaces (in case they are set)
+  # and added as extra attributes to the telemetry in the following format:
+  # k8s.namespace.annotations.<label_name>: <label_value>
+  namespaceAnnotations: []
+
```

Comment:
This setup adds more helm options but with pretty limited flexibility. For example setting it up to collect _all_ annotations/labels will not work because it doesn't set `key_regexp` in `k8sattributes` processor. So we will need to add another helm option to support that. Later one some customers may ask to have custom log attribute names without `k8s.namespace.annotations.` prefix, and we will have to add something else on top of existing helm options. 

What if we expose k8sattributes -> annotations and labels interfaces here as is and users can set it up like

```yaml
extraAttributes
  labels:
    - key_regex: .*
      from: pod 
    - key: release
      from: namespace
  annotations:
    - app-version
      tag_name: app-version
```

It should be pretty easy to use if we provide enough docs and examples here.

`extraAttributes.podLabels` can be deprecated then.

@chaitanyaphalak @rockb1017 WDYT?

Reply:
Oh, i wasn't aware key_regex changed were merged to splunk-otel-collector and were available for use.

In that case, i think your suggestion is better!

Reply:
i like the new approach.

Reply:
Ok cool. Let's do it this way.

Also probably `fromLabels` and `fromAttributes` would sound more intuitive. So it'll be like:

```yaml
extraAttributes:
  fromLabels:
    - key_regex: .*
      from: pod 
    - key: release
      from: namespace
  fromAnnotations:
    - key: app-version
      tag_name: app-version
```